pages/1.11/installing/production/system-requirements/ports/index.md

```diff
@@ -44,35 +48,38 @@ DC/OS allocates additional ports to services running on top of DC/OS. These port
 | 80    | Admin Router Master (HTTP) | `dcos-adminrouter.service` |public IP| master |
 | 443   | Admin Router Master (HTTPS) | `dcos-adminrouter.service`|public IP| master |
-| 1337  | DC/OS Secrets |  `dcos-secrets.service` [enterprise type="inline" size="small" /] | localhost| localhost(master) |
+| 1337  | DC/OS Secrets |  `dcos-secrets.service` | localhost| localhost(master) [enterprise type="inline" size="small" /] |
 | 2181  | ZooKeeper | `dcos-exhibitor.service` | agent/master | master |
 | 3888  | Exhibitor, or ZooKeeper and Exhibitor | `dcos-exhibitor.service` | agent/master | master |
 | 5050  | Mesos Master | `dcos-mesos-master.service` | agent/master | master |
 | 7070  | DC/OS Package Manager (Cosmos) | `dcos-cosmos.service` | localhost| localhost(master) |
 | 8080  | Marathon | `dcos-marathon.service` | agent/master | master |
-| 8101  | DC/OS Identity and Access Manager | `dcos-bouncer.service` [enterprise type="inline" size="small" /] | localhost| localhost(master) |
-| 8123  | Mesos DNS | `dcos-mesos-dns.service` | agent/master | master |
+| 8101  | DC/OS Identity and Access Manager | `dcos-bouncer.service` | localhost| localhost(master) [enterprise type="inline" size="small" /] |
+| 8123  | Mesos DNS | `dcos-mesos-dns.service` | localhost | localhost |
 | 8181  | Exhibitor and ZooKeeper | `dcos-exhibitor.service` | agent/master | master |
-| 8200  | Vault | `dcos-vault.service` [enterprise type="inline" size="small" /] | localhost| localhost(master) |
+| 8200  | Vault | `dcos-vault.service` | localhost| localhost(master) [enterprise type="inline" size="small" /] |
 | 8443  | Marathon SSL | `dcos-marathon.service` | agent/master | master |
-| 8888  | DC/OS Certificate Authority | `dcos-ca.service` [enterprise type="inline" size="small" /] | localhost| localhost(master) |
+| 8888  | DC/OS Certificate Authority | `dcos-ca.service` | localhost| localhost(master) [enterprise type="inline" size="small" /] |
 | 9090 | DC/OS Jobs (Metronome) | `dcos-metronome.service`| agent/master | master |
 | 9443 | DC/OS Jobs (Metronome) SSL | `dcos-metronome.service`| agent/master | master |
 | 9990  | DC/OS Package Manager (Cosmos) | `dcos-cosmos.service` | localhost| localhost(master) |
 | 15055 | DC/OS History | `dcos-history-service.service` | localhost| localhost(master) |
 | 15101 | Marathon libprocess | `dcos-marathon.service` | master | agent/master |
 | 15201 | DC/OS Jobs (Metronome) libprocess | `dcos-metronome.service`| master | agent/master |
-| 62500 | DC/OS Network Metrics | `dcos-networking_api.service` [enterprise type="inline" size="small" /] | master | master |
+| 61053 | Mesos DNS | `dcos-net.service` | agent/master | master [oss type="inline" size="small" /] | 
+| 61430 | DC/OS Net | `dcos-net.service` | agent/master | master [oss type="inline" size="small" /]|
+| 62500 | DC/OS Network Metrics | `dcos-networking_api.service` | master | master [enterprise type="inline" size="small" /] |
```

Comment:
DC/OS Network Metrics was merged into dc/os net in 1.11 as a plugin. it doesn't use port 62500 anymore.